benchto-service/src/main/java/com/teradata/benchto/service/BenchmarkService.java

```diff
@@ -126,7 +127,7 @@ public void startExecution(String uniqueName, String benchmarkSequenceId, String
     @Retryable(value = {TransientDataAccessException.class, DataIntegrityViolationException.class}, maxAttempts = 1)
```

Comment:
Strange indeed. I didn't notice that. Nor did I introduce that. In fact `maxAttempts = 1` is quite popular in this class.